src/styles/Card.scss

```diff
@@ -0,0 +1,102 @@
+$custom-color: #060b35;
+
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  width: 200px;
+  overflow: hidden;
+  box-sizing: border-box;
+
+  &__images {
+    margin-bottom: 40px;
+    text-align: center;
+  }
+
+  &__image {
+    vertical-align: top;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: $custom-color;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: #616070;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__prices {
+    color: #616070;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__money {
+    color: $custom-color;
+    text-align: right;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buy {
+    width: 100%;
+    height: 40px;
+    flex-shrink: 0;
+    border-radius: 5px;
+    background-color: #00acdc;
+    border: none;
+    font-style: normal;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: white;
+    text-transform: uppercase;
+  }
+
+  &__buy:hover {
+    background-color: white;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+  }
+
+  &__reviev {
+    color: #060b35;
+    font-family: Roboto, sans-serif;
```

Comment:
You already set such styling for the `html` tag - it will be automatically applied to all tags and you don't need to repeat it again.
Check that there is no similar duplication